src/api/app/views/webui2/webui/package/_revision_diff_detail.html.haml

```diff
@@ -0,0 +1,14 @@
+%details.details-with-codemirror{ open: expand_diff?(filename, file['state']), id: "revision_details_#{index}" }
+  %summary
+    = filename
+    %span.badge{ class: badge_for_diff_state(file['state']) }= file['state'].capitalize
+    .btn-group.float-right{ role: :group }
+      - unless @linkinfo
+        = link_to_unless(Package.is_binary_file?(filename) || filename.include?('/'),
+        'View file',
+        package_view_file_path(project: project, package: package, filename: filename, rev: calculate_revision_on_state(revision, file['state'])),
+        class: 'btn btn-sm btn-outline-secondary', title: 'View the whole file')
+      - if index.positive?
+        %a.btn.btn-sm.btn-outline-secondary.fas.fa-chevron-up{ href: "#revision_details_#{index - 1}" }
```

Comment:
This is what makes it so weird! I was getting really confused :eyes: This should be `index` or [up](https://github.com/openSUSE/open-build-service/pull/5943/files#diff-6889ea0ae17f916c39b255e1a700bbd0R1) `index-1`